libmbp/devices/jiayu.cpp

```diff
@@ -46,13 +46,29 @@ void addJiayuDevices(std::vector<Device *> *devices)
         "/dev/block/mmcblk0p7" });
     device->setRecoveryBlockDevs({ MTK_RECOVERY, MTK_11230000_RECOVERY,
         "/dev/block/mmcblk0p8" });
-    device->setExtraBlockDevs({ "/dev/block/mmcblk0boot0",
+    device->setExtraBlockDevs({
         MTK_LK, MTK_11230000_LK,
         MTK_LOGO, MTK_11230000_LOGO,
         MTK_PARA, MTK_11230000_PARA,
         MTK_TEE1, MTK_11230000_TEE1,
         MTK_TEE2, MTK_11230000_TEE2,
         MTK_UBOOT, MTK_11230000_UBOOT });
+    device->twOptions()->supported = true;
+    device->twOptions()->flags =
+          Device::FLAG_TW_GRAPHICS_FORCE_USE_LINELENGTH
+        | Device::FLAG_TW_IGNORE_MAJOR_AXIS_0
+        | Device::FLAG_TW_QCOM_RTC_FIX
+        | Device::FLAG_TW_HAS_DOWNLOAD_MODE
```

Comment:
Here only three options I need to test.
FLAG_TW_IGNORE_MAJOR_AXIS_0
FLAG_TW_HAS_DOWNLOAD_MODE
FLAG_TW_QCOM_RTC_FIX